policy/modules/system/userdomain.if

```diff
@@ -605,6 +605,7 @@ template(`userdom_common_user_template',`
 	dev_read_sound($1_t)
 	dev_read_sound_mixer($1_t)
 	dev_write_sound_mixer($1_t)
+	dev_rw_wireless($1_t)
```

Comment:
I'm on the fence. We may want to make a tunable for this.

Reply:
In view of the fact that wireless networks expose to more risks compared to a wired interface coming from a socket in a secured building, it does make sense to make it tunable...

Reply:
I am thinking of calling it "user_manage_rf_devices", but "user_manage_wireless_devices" is also good to me...

Reply:
Actually: "allow_user_manage_rf_devices", defaulting to "true" since the risk is probably medium rather than high, thus giving precedence to usability.

Reply:
I wasn't specifically requesting the change, I was undecided. With what you pointed out, the original change was ok.